### PR TITLE
Update Arcade and run SDL

### PR DESCRIPTION
**Pull request by BrennanConroy**
Update Arcade, so ignore the eng/common files

Call execute-sdl manually instead of running all the arcade jobs that don't apply to this repo.

### CODE REVIEW

File: .azure/default-build.yml [hunk 1]
```diff
@@ -17,8 +17,8 @@ jobs:
     displayName: ${{ coalesce(parameters.jobName, parameters.agentOs) }}
     dependsOn: ${{ parameters.dependsOn }}
     timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
+    enablePublishBuildAssets: false
```
Comment (shirhatti) [inline .azure/default-build.yml:20]: Is this intentional?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -101,41 +96,23 @@ stages:
...
-        -TsaProjectName $(_TsaProjectName)
-        -TsaNotificationEmail $(_TsaNotificationEmail)
-        -TsaCodebaseAdmin $(_TsaCodebaseAdmin)
-        -TsaBugAreaPath $(_TsaBugAreaPath)
-        -TsaIterationPath $(_TsaIterationPath)
-        -TsaRepositoryName "AspNetCore"
-        -TsaCodebaseName "AspNetCore"
-        -TsaPublish $True
-        -PoliCheckAdditionalRunConfigParams @("UserExclusionPath < $(Build.SourcesDirectory)/eng/PoliCheckExclusions.xml")'
+        downloadArtifacts: false
```
Comment (shirhatti) [inline azure-pipelines.yml:118]: Is this intentional?

**Comment by BrennanConroy** [comment]
> Why are you updating arcade manually? No dependency flow?

Correct

File: .azure/default-build.yml (hunk 1 above)
Comment (BrennanConroy) [reply .azure/default-build.yml:20]: We don't have any assets, so yes

**Comment by BrennanConroy** [comment]
No, this is to get successful internal builds because Arcade is mainly made for .NET repos and makes a bunch of assumptions about your build process.

**Pull request merged**